projects/RabbitMQ.Client/client/impl/ModelBase.cs

```diff
@@ -367,20 +367,13 @@ public void HandleCommand(in IncomingCommand cmd)
 
         public void ModelSend(MethodBase method)
         {
-            ModelSend(method, null, ReadOnlyMemory<byte>.Empty);
+            Session.Transmit(new OutgoingCommand(method));
         }
 
         public void ModelSend(MethodBase method, ContentHeaderBase header, ReadOnlyMemory<byte> body)
         {
-            if (method.HasContent)
-            {
-                _flowControlBlock.Wait();
-                Session.Transmit(new OutgoingCommand(method, header, body));
-            }
-            else
-            {
-                Session.Transmit(new OutgoingCommand(method, header, body));
-            }
+            _flowControlBlock.Wait();
```

Comment:
What is the impact of this change due to always waiting on the flow control?

Reply:
It was always called, since this was only called with methods that had HasContent = true. The method above is for non-content methods :)